README.md

```diff
@@ -2,24 +2,13 @@
 
 Master: [![Build Status](https://travis-ci.org/NYPL/nypl-core-objects.svg?branch=master)](https://travis-ci.org/NYPL/nypl-core-objects)
 
-As of version 2.0.0, this module depends on Node 14
+As of version 3.0.0, this module fetches data asyncronously and depends on Node 18.
 
-## The Problem
+Support for v2 is dropped because v2 -> v3 is a easy migration. Apps using version 2x should update to Node18+ and use version 3x.
 
-We have a mapping problem. Here's an example:
-NYPL and [ReCAP](https://recap.princeton.edu/) have their own set of identifiers
-for physical locations here at NYPL.  Different apps refer to these things by different identifiers
-but may want to _convert_ or map them to other known identifiers.
+Apps using version 1x should also update to Node18 and use v3. However, because apps using that version are on much older Nodes, we may update the 1x version of this module for a while - until projects to update those Node versions complete.
```

Comment:
Do you mean that we may continue to maintain the 1x version? A little confused because update is used a lot in this section.

Reply:
I updated the language, hoping to clarify. In general, we need to support versions that are in use. The 1x version is still in use by some apps that are stuck on older Nodes. The 2x version can be phased out because the apps that use it have an easy upgrade path.